.github/workflows/nginx.org-build.yml

```diff
@@ -0,0 +1,76 @@
+name: nginx.org build
+
+on:
+  workflow_call:
+
+permissions:
+  contents: read
+  id-token: write
```

Comment:
My only concern was that this is "write", but after reading https://docs.github.com/en/actions/security-for-github-actions/security-hardening-your-deployments/about-security-hardening-with-openid-connect#adding-permissions-settings it seems we do need "write" here, and there are no other options.